### DISCUSSION TITLE
Nothing Downloading

### DISCUSSION THREAD

**Discussion by PinInTheOcean** [Shabinder/SpotiFlyer#903] [General]
Regardless of what platform I get the music from, SpotiFlyer will no longer download anything. I've tried repairing it and even completely reinstalling it but yet it won't download anything, any idea why or how I can fix it?

**Comment by yavnaz** [comment]
A lot of folks are having the same issue with soundcloud, spotify & youtube. There are many recently opened tickets under the "issues" tab.

**Reply by Shabinder** [reply]
An Update will be available in a day to fix this issue.
Stay Tuned!

**Reply by PinInTheOcean** [reply]
New update giving errors still :(

**Reply by Shabinder** [reply]
Found the cause will be releasing a hotfix

**Reply by Doma777** [reply]
hotfix soon?

**Reply by Shabinder** [reply]
Released Already, v3.6.1 
F-Droid will show in 2-3 days, however is available in Github Releases
